data-sources/codestream/config.yml

```diff
@@ -0,0 +1,17 @@
+id: codestream
```

Comment:
We may or may not want to make this a core data source. I'm working to find out if this should be moved. If so, I'll let you know. We'll need to take this out if that is the case.

Reply:
@josephgregoryii we are going to make this a core data source. Can you remove this data source from this PR?

Reply:
Codestream is now removed!